falcon/response.py

```diff
@@ -1069,5 +1069,6 @@ def __init__(self):
         self.default_media_type = DEFAULT_MEDIA_TYPE
         self.media_handlers = Handlers()
 
-        mimetypes.init()
+        if not mimetypes.inited:
```

Comment:
Curiously, when I run coverage on my box this is the only line that is reported as missing. Apparently the condition is never true (as far as coverage can detect). But I was able to manually verify that the branch does get hit by tests in `asgi.test_asgi_servers`. So something must be beating us to the punch for the in-process tests.

Reply:
Maybe coverage inits the mimetypes? Should me maybe to a reload in the conftest to ensure that we get a clean instance of mimetypes?

Running normally this is hit every time an App is instantiated, so it would be hundreds of times in the falcon tests

This is a workaround for slow windows tests, it's not very important at runtime since the application is not continually instantiated. So we could place an `is_windows and falcon_tests` here.

Reply:
slow windows test may happen on users test suite, since it is a similar situation regarding instantiating the app many times

Reply:
I've added a test for it in the last commit